ddl/index.go

```diff
@@ -489,7 +486,21 @@ func (w *worker) onCreateIndex(d *ddlCtx, t *meta.Meta, job *model.Job, isPK boo
 		indexInfo.Global = global
 		indexInfo.ID = allocateIndexID(tblInfo)
 		tblInfo.Indices = append(tblInfo.Indices, indexInfo)
-
+		if tblInfo.IsCommonHandle && !indexInfo.Primary {
```

Comment:
There are several codes to check the index length in this PR, but the conditions are different. I'm a little dizzy. Could you unify one function? Or add a note? I don't quite understand why this check is added here